diagnostic.md

```diff
@@ -5,74 +5,85 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+to manage data (store created and updated data, send and receive data)
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+they are outdated, we don't need no views.
```

Comment:
Well partly true, we do want to utilize views but now we do it on the front-end.